examples/canvg_context2d/bar_graph_with_text_and_lines.html

```diff
@@ -840,16 +839,13 @@ <h1>Bar Graph With Text And Lines</h1>
                         ctx.fillRect(0, 0, 1000, 700);
 
                         //load a svg snippet in the canvas with id = 'drawingArea'
-                        canvg(c, document.getElementById('svg').outerHTML, {
-                              ignoreMouse: true,
-                              ignoreAnimation: true,
-                              ignoreDimensions: true
-                        });
-
+                        const v = await canvg.Canvg.from(ctx, document.getElementById('svg').outerHTML, canvg.presets.offscreen());
+                        await v.render();
                         return pdf;
                   };
-                  document.getElementById('result').setAttribute('src', makePdf().output('dataurlstring'));
-                  document.getElementById('source').innerText = makePdf().output();
+                  const pdf = await makePdf();
+                  document.getElementById('result').setAttribute('src', pdf.output('dataurlstring'));
+                  document.getElementById('source').innerText = pdf.output();
```

Comment:
You may call `output()` only once per document.

Reply:
good to know, this is now fixed